applications/crossbar/src/modules/cb_ledgers.erl

```diff
@@ -247,16 +252,19 @@ put(Context, Action) ->
            ,kz_json:get_ne_json_value(<<"metadata">>, ReqData, kz_json:new())
            }
           ,{fun kz_ledger:set_dollar_amount/2
-           ,abs(kz_json:get_integer_value(<<"amount">>, ReqData, 0))
+           ,Dollars
+           }
+          ,{fun kz_ledger:set_unit_amount/2
```

Comment:
@jamesaimonetti why set both units and dollars?

Reply:
Because we have them...